tutor/commands/k8s.py

```diff
@@ -225,10 +234,16 @@ def launch(context: click.Context, non_interactive: bool) -> None:
             from_release=tutor_env.get_env_release(context.obj.root),
         )
 
-    click.echo(fmt.title("Interactive platform configuration"))
     config = tutor_config.load_minimal(context.obj.root)
     if not non_interactive:
-        interactive_config.ask_questions(config, run_for_prod=True)
+        click.echo(fmt.title("Interactive platform configuration"))
+        interactive_config.ask_questions(
+            config, context.obj.root, run_for_prod=True, clean_environment=clean_env
+        )
+    else:
```

Comment:
I'm not 100% sure we want to support a `launch --clean` option. What do ya'll think?

Reply:
I'm okay with removing this but it's the shorthand for:

`tutor config save --clean && tutor dev|local|k8s launch`

Reply:
We're adding a simple boolean command flag and it ends up affecting 6 different modules and functions. The fact that the `clean` logic has to be handled both by `launch` and `save` makes the feature too complex, IMHO. Since this option is not recommended by default, I would suggest to add it only to the `config save` command, which will make the code much simpler.

Reply:
Regis makes good points, I am fine with the `launch -c` flag being removed.

Reply:
OK. I removed the launch flag.